### PR TITLE
Feat/4251  rename activity feed and add publication route

### PR DESCRIPTION
**Pull request by asf-algoo**
https://github.com/tracim/tracim/issues/4251

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by Skylsmoi** [comment]
No unit tests added since this PR is mostly a renaming.
Tests have only been updated

File: frontend/i18next.scanner/en/translation.json [hunk 1]
```diff
@@ -411,5 +409,8 @@
   "Account restored": "Account restored",
   "New access": "New access",
   "Access removed": "Access removed",
-  "Requested access": "Requested access"
+  "Requested access": "Requested access",
+  "Recent activities": "Recent activities",
+  "Loading recent activities…": "Loading recent activities…",
+  "undeleted": "undeleted"
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/en/translation.json]:
```suggestion
  "undeleted": "restored"
```

File: frontend/i18next.scanner/fr/translation.json [hunk 2]
```diff
@@ -411,5 +409,8 @@
   "Account restored": "Compte restauré",
   "New access": "Nouvel accès",
   "Access removed": "Accès retiré",
-  "Requested access": "Accès demandé"
+  "Requested access": "Accès demandé",
+  "Recent activities": "Activités récentes",
+  "Loading recent activities…": "Chargement des activités récentes…",
+  "undeleted": "Restauré"
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/fr/translation.json]:
```suggestion
  "undeleted": "restauré"
```

File: frontend/i18next.scanner/en/translation.json [hunk 3]
```diff
@@ -411,5 +409,9 @@
   "Account restored": "Account restored",
   "New access": "New access",
   "Access removed": "Access removed",
-  "Requested access": "Requested access"
+  "Requested access": "Requested access",
+  "Recent activities": "Recent activities",
+  "Loading recent activities…": "Loading recent activities…",
+  "undeleted": "undeleted",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/en/translation.json:415]:
```suggestion
  "undeleted": "restored",
```

File: frontend/i18next.scanner/fr/translation.json [hunk 4]
```diff
@@ -411,5 +409,9 @@
   "Account restored": "Compte restauré",
   "New access": "Nouvel accès",
   "Access removed": "Accès retiré",
-  "Requested access": "Accès demandé"
+  "Requested access": "Accès demandé",
+  "Recent activities": "Activités récentes",
+  "Loading recent activities…": "Chargement des activités récentes…",
+  "undeleted": "Restauré",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/fr/translation.json:415]:
```suggestion
  "undeleted": "restauré",
```

File: frontend/i18next.scanner/en/translation.json (hunk 3 above)
Comment (Skylsmoi) [reply frontend/i18next.scanner/en/translation.json]: fixed

File: frontend/i18next.scanner/fr/translation.json (hunk 4 above)
Comment (Skylsmoi) [reply frontend/i18next.scanner/fr/translation.json]: fixed

**Review by gbonaspetti — APPROVED** [review]

**Comment by gbonaspetti** [comment]
Tested on branch

**Pull request merged**